docs/KARAVICTL_GUIDE.md

```diff
@@ -114,15 +114,15 @@ karavictl inject [flags]
 Inject into an existing vxflexos CSI driver
 ```
 kubectl get secrets,deployments,daemonsets -n vxflexos -o yaml \
-   | karavictl inject --image-addr [IMAGE_REPO]:5000/sidecar-proxy:latest --proxy-host [PROXY_HOST_IP] \
+   | karavictl inject --image-addr [IMAGE_REPO]:5000/sidecar-proxy:1.0.0 --proxy-host [PROXY_HOST_IP] \
```

Comment:
No changes required here.  Content has moved to the common doc site so please make changes there instead

Reply:
removed